inc/Styles/Component.php

```diff
@@ -174,22 +174,15 @@ public function action_preload_styles() {
 	}
 
 	/**
-	 * Enqueues WordPress theme styles for the block editor.
+	 * Enqueues WordPress theme styles for the editor.
 	 */
-	public function action_enqueue_block_editor_styles() {
+	public function action_add_editor_styles() {
 
 		// Enqueue Google Fonts.
-		$google_fonts_url = $this->get_google_fonts_url();
-		if ( ! empty( $google_fonts_url ) ) {
```

Comment:
This check needs to be maintained. Only if Google fonts URL is not empty, it should be added as editor style.

Reply:
Thanks